src/main/kotlin/uk/gov/justice/digital/hmpps/nomisprisonerapi/data/CreateVisitRequest.kt

```diff
@@ -52,4 +52,8 @@ data class CreateVisitRequest(
   @Schema(description = "Location of visit, an agency internal location code", required = true)
   @NotBlank
   val visitRoomId: String,
+
+  @Schema(description = "VSIP visit id to allow mapping of nomis to VSIP visits", required = true)
+  @NotBlank
+  val vsipVisitId: String,
```

Comment:
Interesting one .. just thinking about whether this should be in the url: on the one hand a creation post rest call should probably not have an id in the url, but on the other it would be consistent with the cancel which will do

Reply:
hmmm, I think it works for the cancel but its part of the body to persist for the create, rather than an identifier